src/components/textField/index.tsx

```diff
@@ -74,7 +76,7 @@ const TextField = (props: InternalTextFieldProps) => {
     enableErrors, // TODO: rename to enableValidation
     validationMessageStyle,
     validationMessagePosition = ValidationMessagePosition.BOTTOM,
-    retainValidationSpace = true,
+    retainValidationSpace = !helperText,
```

Comment:
Just to make sure I understand
because helperText now renders under the input then we don't need to retain space if it exists?

Reply:
Yes. Otherwise there will be a gap btw the input and the helperText